content/en/blogs/kubesphere-3.1.0-ga-announcement.md

```diff
@@ -0,0 +1,156 @@
+
+"KubeSphere has emerged as one of the most popular Kubernetes distributions," said Ray Zhou, Head of the KubeSphere project. "The release of KubeSphere 3.1 marks an important milestone as we further empower enterprises to gain first-mover advantages especially in edge computing."
+
+In a world where everything is connected, our legacy system with centralized storage and computing can no longer satisfy our demand for efficiency and capacity. An increasing number of use cases requiring low-latency processing, AI and IoT have all spoken volumes for a booming edge computing market. According to [a CNCF survey](https://www.cncf.io/wp-content/uploads/2021/05/KubernetesEdge_Survey_Report_2021_v2.pdf), the top three challenges facing enterprises when authoring, deploying and/or running applications at the edge are:
+
+1. Security of the deployment and content.
+2. Edge devices going offline.
+3. Observability of activities/status at edge.
+
+Against this backdrop, the KubeSphere community has been working closely with the KubeEdge community to take Kubernetes to the edge. With a unified standard, users now can add and manage edge nodes to their cluster, deploy workloads on edge nodes, and monitor edge nodes on the back of a strong observability system.
+
+And most importantly, the new version would not have been possible without the contribution of every member in the community, whose footprints can be found in virtually every aspect, including feature development, function testing, bug reports and fixes, requests and suggestions, enterprise best practices, internationalization and localization, and documentation. All of these explain the power of the community and what the community will continue to offer for a long time to come.
+
+## Major Updates at a Glance
+
+![notable-changes-in-3.1](/images/blogs/en/kubesphere-3.1.0-ga-announcement/notable-changes-in-3.1.png)
+
+### Metering and billing
+
+As part of our commitment to providing a holistic platform, KubeSphere v3.1.0 is equipped with a brand-new feature of [metering and billing](https://kubesphere.io/docs/toolbox/metering-and-billing/view-resource-consumption/). It records resource usage in clusters and workspaces while you can dive deep into nodes, projects, workloads, and Pods.
+
+You use KubeSphere **Metering and Billing** to:
+
+- Track detailed resource consumption at different levels on a unified dashboard.
+- Specify a date range to view data within a specific billing cycle.
+- Customize the prices of multiple resources, such as CPU, memory and VPCs.
```

Comment:
`VPCs` are not that exact from my opinion, I assume we can use `network` or `traffic` here.

@zheng1 Please help us to verify it.

Reply:
![image](https://user-images.githubusercontent.com/65327072/118427551-14190880-b700-11eb-9f20-decc05aa133b.png)
If you think this is not accurate, perhaps I can use `storage` instead.